### PR TITLE
feat(host): add link name to wRPC invocations

### PR DESCRIPTION
**Pull request by vados-cosmonic**
## Feature or Problem

This commit adds the "link-name" header to invocations performed by the host using wRPC.

Resolves https://github.com/wasmCloud/wasmCloud/issues/2332

### CODE REVIEW

File: crates/host/src/wasmbus/handler.rs [hunk 1]
```diff
@@ -103,63 +107,64 @@ impl Handler {
 
     #[instrument(level = "trace", skip(self))]
     async fn wrpc_blobstore_blobstore(&self) -> anyhow::Result<wasmcloud_core::wrpc::Client> {
-        let LatticeInterfaceTarget { id, .. } = self
+        let lit = self
```
Comment (thomastaylor312) [inline crates/host/src/wasmbus/handler.rs:110]: tenor-258100641

**Review by thomastaylor312 — APPROVED** [review]

File: crates/host/src/wasmbus/handler.rs [hunk 2]
```diff
@@ -103,63 +107,64 @@ impl Handler {
...
             .identify_wrpc_target(&CallTargetInterface::from_parts((
                 "wasi",
                 "blobstore",
                 "blobstore",
             )))
             .await
             .context("unknown `wasi:blobstore/blobstore` target")?;
-        Ok(self.wrpc_client(&id))
+        Ok(self.wrpc_client(&lit))
```
Comment (brooksmtownsend) [inline crates/host/src/wasmbus/handler.rs:118]: tenor-124450742

File: crates/host/src/wasmbus/handler.rs (hunk 1 above)
Comment (vados-cosmonic) [reply crates/host/src/wasmbus/handler.rs:110]: :fire: :fire: :fire:

**Review by brooksmtownsend — APPROVED** [review]
Well that ended up being a nice change!

**Pull request merged**